From 88c6180c807bcdcbe5a189fa7a2a7432cd3a750d Mon Sep 17 00:00:00 2001
From: Boris Ranto <branto@redhat.com>
Date: Fri, 6 Nov 2015 12:06:52 +0100
Subject: [PATCH] ipmitool-1.8.10-ipmievd-init.patch

562186 - Initscript of the ipmievd service not LSB compliant

Backported by: dgregor jsafrane
---
 contrib/ipmievd.init.redhat | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/contrib/ipmievd.init.redhat b/contrib/ipmievd.init.redhat
index 1c0253d..7cf9d24 100755
--- a/contrib/ipmievd.init.redhat
+++ b/contrib/ipmievd.init.redhat
@@ -5,7 +5,7 @@
 #       Based on example sysvinitfiles script
 #       Copyright (c) 2000 Red Hat Software, Inc.
 #
-# chkconfig: 345 99 00
+# chkconfig: - 99 00
 # description: ipmievd daemon to send events to syslog
 # processname: ipmievd
 # config:      /etc/sysconfig/ipmievd
@@ -16,8 +16,8 @@
 # Should-Start: $time
 # Required-Stop:     $syslog ipmi
 # Should-Stop: $time
-# Default-Start:     3 4 5
-# Default-Stop:      0 1 2 6
+# Default-Start:     
+# Default-Stop:      
 # Short-Description: ipmievd daemon to send events to syslog
 # Description:       Start ipmievd to read events from BMC and
 #	             log them to syslog.  Events correspond to hardware faults,
@@ -81,7 +81,7 @@ case "$1" in
 	;;
     *)
 	echo "Usage: ipmievd {start|stop|status|reload|restart|condrestart}"
-	exit 1
+	exit 2
 	;;
 esac
 exit $?
-- 
2.1.0

